lib/PuppeteerSharp/PuppeteerSharp.csproj

```diff
@@ -34,7 +34,8 @@
     <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
     <PackageReference Include="System.Net.Http" Version="4.3.3" />
     <PackageReference Include="Microsoft.AspNetCore.WebUtilities" Version="2.0.2" />
-    <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.1" />
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="1.1.0" />
+    <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.2" />
```

Comment:
This version bumped because it is a dependency from AsyncInterfaces. If you'd like, we could now drop the explicit reference to System.Threading.Tasks.Extensions and just consume it transitively through AsyncInterfaces.